src/systems/filecoin_blockchain/struct/block/_index.md

```diff
@@ -13,5 +13,71 @@ A block header contains information relevant to a particular point in time over
 
 {{< readfile file="block.id" code="true" lang="go" >}}
 
-{{< readfile file="block.go" code="true" lang="go" >}}
+{{< readfile file="election.id" code="true" lang="go" >}}
+
+# Block syntax validation
+
+Syntax validation refers to validation that may be performed on a block and its messages 
+without reference to outside information such as the parent state tree.
+
+An invalid block must not be transmitted or referenced as a parent.
+
+A syntactically valid block header must decode into fields matching the type definition below. 
+
+A syntactically valid header must have:
+
+- a non-empty array of `Parents` CIDs if `Epoch` is greater than zero,
+- a non-negative `ParentWeight`,
+- a `Miner` address which is an ID-address,
+- a non-negative `Epoch`,
+- a positive `Timestamp`,
+- a `Ticket` with non-empty `VRFResult`,
+- `ElectionPoStOutput` containing:
+   - a non-empty `Candidates` array
+   - a non-empty `PoStRandomness` field
+   - a non-empty `Proof` field
+   
+A syntactically valid full block must have:
+
+- all referenced messages syntactically valid,
+- all referenced parent receipts syntactically valid,
+
+Note that validation of the block signature requires access to the miner worker address and
+public key from the parent tipset state, so signature validation forms part of semantic validation. 
+Similarly, message signature validation requires lookup of the public key associated with 
+each message's `From` account actor in the block's parent state.
+
+# Block semantic validation
+
+Semantic validation refers to validation that requires reference to information outside the block
+header and messages themselves, in particular the parent tipset and state on which the block is built.
+
+A semantically valid block must have:
+
```

Comment:
Since we are planning to remove duplication by putting these all in one place (https://github.com/filecoin-project/specs/pull/709), we need a way to reference these rules succinctly from the chain sync section.  The `BV*` method used in the chain sync doc worked ok.  Consider reinstating this grouping.  If you want to avoid grouping here then each bullet here could be a number / BV string and we could do the grouping in the chain sync doc.

Reply:
I didn't see #709 and I don't think the progressive notes belong here. I have moved them back.

I see a clear distinction between the prescriptive validation rules described here, and the recommendation of progressive validation in the chain sync process.